lib/logging.js

```diff
@@ -0,0 +1,56 @@
+
+'use strict';
+
+var path = require('path');
+var eachModule = require('each-module');
+var winston = require('winston');
+
+module.exports = function (config) {
+	// User-defined configurations take priority, but fallback to defaults if all seem invalid
+	var getArrayOfValidModulesByDirName = function (finalDir) {
+		var modules = [];
+		var modulePusher = function (moduleName, moduleExports, file) {
+			if (typeof moduleExports === 'function') {
+				modules.push(moduleExports);
+			} else {
+				console.warn('Invalid module dropped ' + file);
+			}
+		};
+
+		var locations = [config.path.root, config.path.shunterRoot]; // Config.path.root = users files
```

Comment:
`root` vs `shunterRoot` makes no sense to me, the former is too ambiguous, it tripped me up whilst devving :)